### DISCUSSION TITLE
You are currently using 88.54 MB (92.8%) of your 95 MB.

### DISCUSSION THREAD

**Discussion by dyl188** [chamilo/chamilo-lms#5757] [General]
When creating a course chapter, I receive a "disk space insufficient" error when uploading a video file. How can I modify the disk space quota?
![微信截图_20240828113611](https://github.com/user-attachments/assets/8eef32fd-6bfd-4640-a4c6-019f8622c5a7)

**Comment by Coursenligne** [comment]
Go to Administration tab, then courses.
Edit the course parameters and you can change the default value of 95 MB.
Note that you have to ensure the settings of your server as larges files may caus issues...

**Reply by dyl188** [reply]
Thank you very much
